Src/FluentAssertions/Primitives/HttpResponseMessageAssertions.cs

```diff
@@ -0,0 +1,222 @@
+﻿using System.Diagnostics;
+using System.Net;
+using System.Net.Http;
+using FluentAssertions.Execution;
+
+namespace FluentAssertions.Primitives
+{
+    /// <summary>
+    /// Contains a number of methods to assert that a <see cref="HttpResponseMessage"/> is in the expected state.
+    /// </summary>
+    [DebuggerNonUserCode]
+    public class HttpResponseMessageAssertions
+        : HttpResponseMessageAssertions<HttpResponseMessageAssertions>
+    {
+        public HttpResponseMessageAssertions(HttpResponseMessage value)
+            : base(value)
+        {
+        }
+    }
+
+    /// <summary>
+    /// Contains a number of methods to assert that a <see cref="HttpResponseMessage"/> is in the expected state.
+    /// </summary>
+    [DebuggerNonUserCode]
+    public class HttpResponseMessageAssertions<TAssertions>
```

Comment:
This should probably either derive from `ReferenceTypeAssertions` or `ObjectAssertions`.
Deriving from `ObjectAssertions` will give the least invasive changes, as `HttpResponseMessage` currently should-resolves to `ObjectAssertions`.